Jenkinsfile

```diff
@@ -69,6 +69,39 @@ pipeline {
         stage('OIDC Authenticator') {
           steps { sh 'ci/test cucumber_authenticators_oidc' }
         }
+        stage('Azure Authenticator preparation - Allocate Azure Authenticator Instance') {
```

Comment:
This pattern is unfamiliar, so probably worth adding a comment about why the two stages are necessary.

Reply:
Thanks, added a comment

Reply:
I would mention why this is a special case. Like why don't we have this for OIDC for example? That way it is more clear that this is pattern circumstantial

Reply:
```
// We have 2 stages for the Azure Authenticator tests. One is
// responsible for allocating an Azure VM, from which we will get an
// Azure access token for Conjur authentication. It sets the Azure VM IP
// in the env and waits until the authn-azure tests are finished. We use
// this mechanism as we need a live Azure VM for getting the Azure token
// and we don't want to have a long running-machine deployed in Azure.
```

@sigalsax what do you think? I believe the last sentence is redundant. I can't say why we don't have it for OIDC because we just don't need it there.